js/setup.js

```diff
@@ -0,0 +1,89 @@
+"use strict";
+
+const NAMES = [
+  `Иван`,
+  `Хуан Себастьян`,
+  `Мария`,
+  `Кристоф`,
+  `Виктор`,
+  `Юлия`,
+  `Люпита`,
+  `Вашингтон`
+];
+const SURNAMES = [
+  `да Марья`,
+  `Верон`,
+  `Мирабелла`,
+  `Вальц`,
+  `Онопко`,
+  `Топольницкая`,
+  `Нионго`,
+  `Ирвинг`
+];
+const COAT_COLOR = [
+  `rgb(101, 137, 164)`,
+  `rgb(241, 43, 107)`,
+  `rgb(146, 100, 161)`,
+  `rgb(56, 159, 117)`,
+  `rgb(215, 210, 55)`,
+  `rgb(0, 0, 0)`,
+];
+const EYES_COLOR = [
+  `black`,
+  `red`,
+  `blue`,
+  `yellow`,
+  `green`
+];
+const NUMBER_PLAYERS = 4;
+
+const setup = document.querySelector(`.setup`);
```

Comment:
```suggestion
const setupNode = document.querySelector(`.setup`);
```
Хорошая практика использовать слово `node` в нейминге, когда идет работа с DOM узлами. Также в нейминге хорошо упоминать в постфиксе, что возвращает функция или что переменная в себе хранит пр.: `...arr`, `...int`, `...str`, `...obj`. 

Так в будущем будет удобнее с ними работать, понимая из названия что внутри сущности. Такой код легче дебажить и можно избежать глупых ошибок _(JS не строго типизирован, потому часто бывает неочевидное поведение из-за приведения типов)_.